File: webscale-classic/src/main/java/com/eforce21/webscale/classic/controller/ImageGetParams.java
package com.eforce21.webscale.classic.controller;

public class ImageGetParams {

    private Integer pageNumber;
    private Integer amount;

    private String filterString;

    public Integer getPageNumber() {
        return pageNumber;
    }

    public void setPageNumber(Integer pageNumber) {
        this.pageNumber = pageNumber;
    }

    public Integer getAmount() {
        return amount;
    }

    public void setAmount(Integer amount) {
        this.amount = amount;
    }

    public String getFilterString() {
        return filterString;
    }

    public void setFilterString(String filterString) {
        this.filterString = filterString;
    }
}

File: webscale-ui/src/app/images/model/image-get-params.ts
export class ImageGetParams {

    private _pageNumber: number;
    private _amount: number;
    private _filterString: string;

    get pageNumber(): number {
        return this._pageNumber;
    }

    set pageNumber(value: number) {
        this._pageNumber = value;
    }

    get amount(): number {
        return this._amount;
    }

    set amount(value: number) {
        this._amount = value;
    }

    get filterString(): string {
        return this._filterString;
    }

    set filterString(value: string) {
        this._filterString = value;
    }
}